src/main/java/com/tournament/cup/play/TeamsListEight.java

```diff
@@ -0,0 +1,53 @@
+package com.tournament.cup.play;
+
+import com.tournament.cup.details.Squad;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeamsListEight {
+    List<Squad> squads = new ArrayList<>();
```

Comment:
This is stateful component(it has mutable property). By default all spring components are singletons which means that this won't work well in concurrent environment. For this CLI app it's not a problem, but for future web projects don't use non-synchronized properties in singletons